docs/data/scheduler/datasets/personal-agenda.ts

```diff
@@ -200,6 +200,14 @@ export const initialEvents: CalendarEvent[] = [
     title: 'Shopping',
     resource: 'personal',
   },
+  {
```

Comment:
Not strictly related to this PR, but I would like to remove the all day events that don't represent actual data (and maybe create new ones, the birthdays should be all day for example).

Do you think we should have a single "big data" data set for both recurring event and all day events (in which case we need to at least rename your new data set) or should we keep them separate (in which case you don't have to do anything, we can just create the new all day events dataset in a follow up)